stable/kube-state-metrics/values.yaml

```diff
@@ -94,6 +94,8 @@ collectors:
   secrets: true
   services: true
   statefulsets: true
+  ingresses: true
```

Comment:
Let's continue following the alphabetical order here.

Reply:
Done. Sorry, didn't realize about the order from the first sight.